### ISSUE TITLE
does not seem to work

### ISSUE THREAD

**Issue by despairblue** [demoneaux/reveal-code-focus#5]
When I have something linke this on the page:

![screen shot 2015-11-20 at 23 32 50](https://cloud.githubusercontent.com/assets/927609/11313727/691b3bda-8fdf-11e5-84cb-05b085bb2d43.png)

It will be this after calling `RevealCodeFocus()`:

![screen shot 2015-11-20 at 23 33 07](https://cloud.githubusercontent.com/assets/927609/11313738/78d9663c-8fdf-11e5-84e2-67f944fb12ff.png)

I also tried the example in the README to make sure that this does not happen because it is XML. Though that should not be the problem, since it will be escaped before that happens:

```javascript
			// Split highlighted code into lines.
			element.innerHTML =
				element.innerHTML
				.replace(/^(<[^>]+>)?/, function(_, html) {
					if (html && ~html.indexOf('hljs-comment')) {
						console.log(html.indexOf('comment'))
						console.log(html)
						return html + '<span class=line>';
					} else {
						return '<span class=line>' + (html ? html : '');
					}
				})
				.replace(/\n\n/g, function() {
					return '\n&nbsp;\n';
				})
				.replace(/\n/g, '</span><span class=line>') + '</span>';
```

I can't see the problem at first glance.

**Comment by d10** [comment]
The code is currently slightly hacky, as I'm just wrapping each line with a `<span>` element. I'm not too sure what's happening for now, will try to check and solve soon.

**Comment by Akuryou** [comment]
Got the same issue with a chunk of JSON.

**Comment by demoneaux** [comment]
I'm mostly done with an improved solution to obtaining lines. @Akuryou and @despairblue if you could provide me with any code samples that don't work that would be great for adding tests.

**Comment by Akuryou** [comment]
Hi,
i pushed a branch that uses code-focus:
https://github.com/Akuryou/introduction-to-swagger/tree/code-focus

I added some sample fragments on slide 10 that produce the error.

**Comment by demoneaux** [comment]
This should be fixed by dd9b17f18e5748eb0bf3331e88141c43c8df9e7a, which is available in 2.0. @Akuryou I'll also leave some comments in your code-focus branch.

**Comment by Akuryou** [comment]
YES! That fixed it.
Nice job!
